vendor/models/invoice.py

```diff
@@ -267,6 +267,11 @@ def save_discounts_vendor_notes(self):
         self.vendor_notes["discounts"] = self.get_discounts()
         self.save()
 
+    def get_promos(self):
+        if 'promos' not in self.vendor_notes or len(self.vendor_notes.get('promos', 0)):
```

Comment:
this second check should probably be something like len(self.vendor_notes["promos"]) == 0; i dont think the current check will give the correct result

Reply:
Thanks, this only shows how important are unit test jajajajaj. Thanks @mgcamisa